docs/WSL.md

```diff
@@ -21,31 +21,16 @@ here: https://channel9.msdn.com/Shows/Tabs-vs-Spaces/GPU-Accelerated-Machine-Lea
 
 Requirements: Windows 10 version 2004 or higher
 
-The instructions are [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10), but summarized in
-copy/paste-able form below. When installing via the UI, pick Ubuntu version 20.04 LTS as your distribution.
-
 To use the commandline setup, please first install
```

Comment:
The step to install Windows Terminal should be moved a bit lower. Also, it could be made clear that this is optional. People can then either start Ubuntu directly, or start a window in the terminal. (this should be clear before reaching line 50)

Reply:
Just moved the section lower, and committed the changes!